src/runtime/hexagon_remote/bin/.clang-format

```diff
@@ -1 +0,0 @@
-DisableFormat: true
```

Comment:
Please undelete this file: we don't want clang-format to be applied to the stuff in hexagon_remote/src